test/blackbox-tests/test-cases/coq/coqpp.t/run.t

```diff
@@ -0,0 +1,70 @@
+Testing the coqpp stanza
+
+  $ cat > dune << EOF
+  > (coq.pp
+  >  (modules gram))
+  > (library
+  >  (name foo))
+  > EOF
+
+  $ ls
+  dune
+  dune-project
+  gram.mlg
+  marg.mlg
+
+  $ dune build
+  File "gram.ml", line 2, characters 8-30:
+  2 | let _ = Mltop.add_known_module __coq_plugin_name
```

Comment:
@ejgallego Do you know what we can do about this?

Reply:
You need to depend on `coq-core.vernac` on the library stanza, no other way around that as `coqpp` will generate Coq-specific code.

Reply:
Thanks